packages/lesswrong/lib/betas.ts

```diff
@@ -74,6 +74,12 @@ export const useRecombeeFrontpage = (currentUser: UsersCurrent|DbUser|null) => {
   return isLW && (isAdmin(currentUser) || manualOptIn) && recombeeEnabledSetting.get()
 }
 
+export const userHasLlmChat = (currentUser: UsersCurrent | DbUser): boolean => {
+  const userIdsWithAccess = userIdsWithAccessToLlmChat.get();
+
+  return isLW && (isAdmin(currentUser) || userIdsWithAccess.includes(currentUser?._id));
```

Comment:
No point in doing a null-safety check if `currentUser` can't be null or undefined.  (But you probably should let `currentUser` be null in the function signature; that's what we do elsewhere and it's less annoying to use `userHasLlmChat` if you don't need to check whether `currentUser` exists first.)